ostd/src/mm/frame/meta.rs

```diff
@@ -152,9 +152,12 @@ pub unsafe trait AnyFrameMeta: Any + Send + Sync + Debug + 'static {
 macro_rules! impl_frame_meta_for {
     // Implement without specifying the drop behavior.
     ($t:ty) => {
-        use static_assertions::const_assert;
-        const_assert!(size_of::<$t>() <= $crate::mm::frame::meta::FRAME_METADATA_MAX_SIZE);
-        const_assert!(align_of::<$t>() <= $crate::mm::frame::meta::FRAME_METADATA_MAX_ALIGN);
+        static_assertions::const_assert!(
```

Comment:
The `static_assertions` crate is deprecated. Use the `const_assert` macro from [ostd/src/util.rs](https://github.com/asterinas/asterinas/blob/3d21258baf3dd89cb4e9d44d6da8c4c8190478a4/ostd/src/util.rs#L15) instead.